src/store/metadata/stub/content/editions/versions/local-authority-greenhouse-gas-emissions_2023-06.json

```diff
@@ -0,0 +1,76 @@
+{
+    "@context": "https://staging.idpd.uk/ns#",
+    "@id": "https://staging.idpd.uk/datasets/local-authority-greenhouse-gas-emissions/editions/2023-06/versions",
+    "@type": "hydra:Collection",
+    "title": "Local authority greenhouse gas emissions",
+    "versions": [
+        {
+            "@type": [
+                "dcat:Distribution",
+                "csvw:Table"
+            ],
+            "@id": "https://staging.idpd.uk/datasets/local-authority-greenhouse-gas-emissions/editions/2023-06/versions/1",
+            "identifier": "1",
+            "issued": "2023-06-29T00:00:00Z",
+            "title": "Local authority greenhouse gas emissions",
+            "summary": "UK local authority estimates of greenhouse gas emissions",
+            "description": "These statistics provide a breakdown of greenhouse gas emissions across the UK, using...",
+            "download_url": "https://staging.idpd.uk/datasets/local-authority-greenhouse-gas-emissions/editions/2023-06/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+                "columns": [
+                    {
+                      "name": "year",
+                      "datatype": "datetime",
+                      "titles": "Year",
+                      "description": "The year associated with the observation."
+                    },
+                    {
+                      "name": "country",
+                      "datatype": "decimal",
```

Comment:
The datatypes in this file appear to be wrong. What should be strings are decimals. Aside from year, all other columns use decimal as their datatype.